src/main/java/multithreading/Race.java

```diff
@@ -0,0 +1,10 @@
+package multithreading;
+
+public class Race {
+
+    public static void main(String[] args) {
+        new ThreadExt().start();
+        new RunnableImpl().run();
```

Comment:
in this case we run thread (RunnableImpl) in main thread. Do you know smth about that?

Reply:
Replaced "new RunnableImpl().run()" with "new Thread(new RunnableImpl(counter)).start()"